mne/viz/raw.py

```diff
@@ -445,6 +445,7 @@ def plot_raw(raw, events=None, duration=10.0, start=0.0, n_channels=20,
     params['callback_key'] = callback_key
     # have to store this, or it could get garbage-collected
     params['opt_button'] = opt_button
+    params['plot_vertline'] = partial(_draw_vert_line, params=params)
```

Comment:
You don't seem to connect the _draw_vert_line function to 'button_press_event' but modify `_mouse_click`. Maybe it will be clearer if you attach another function to `'button_press_event'`?

Reply:
ok, I now see that _mouse_click aggregates different mouse functionalities, so although I still think attaching multiple functions might be clearer it goes against how this code is composed.